external/concealer/include/concealer/autoware_def.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015-2021 Tier IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef CONCEALER__AUTOWARE_DEF_HPP_
+#define CONCEALER__AUTOWARE_DEF_HPP_
+
+#define AUTOWARE_AUTO
```

Comment:
NOTE: This code does not support both Auto and ArchitectureProposal at the same time.

Changing the default Autoware version to Auto will affect the existing Tier IV CI / CD pipeline (using scenario_simulator_v2 for testing).
I want ArchitectureProposal to be the default until dynamic Autoware architecture selection is supported.